<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Encryption</class>
 <widget class="QWidget" name="Encryption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </item>
     <item row="1" column="1">
      <widget class="QGroupBox" name="gb_edchoice">
       <property name="title">
        <string/>
       </property>
       <widget class="QRadioButton" name="rb_encrypt">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>69</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Encrypt</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="rb_decrypt">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>0</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Decrypt</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Key:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_key">
         <property name="placeholderText">
          <string>e.g. (Apple, 0111, F08A)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <widget class="QRadioButton" name="rb_ascii">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>55</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>ASCII</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="rb_binary">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>0</y>
          <width>63</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Binary</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="rb_hex">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>0</y>
          <width>48</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Hex</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
